Makefile

```diff
@@ -1,10 +1,9 @@
-# Image URL to use all building/pushing image targets
-IMG ?= europe-docker.pkg.dev/kyma-project/prod/telemetry-manager:main
-# ENVTEST_K8S_VERSION refers to the version of Kubebuilder assets to be downloaded by envtest binary.
+include .env
+export $(shell sed 's/=.*//' .env)
```

Comment:
Can you explain what is the purpose of this command?

Reply:
This is the actual command that searches for and exports all environment variables from the .env file, it does the following:
- `sed`: UNIX stream editor command, used for searching, replacing, deleting content that matches a specific pattern
- `'s/=.*//'`: the actual search pattern, that matches the variable names and exports them
- `.env`: the given file where to search for

To be honest, I've just checked it and it should work perfectly fine without this command as well. I guess it was just how people use it, in order to actually export those variables as environment variables. If we don't need it, I can remove it.